src/ocamlorg_frontend/css/partials/package_breadcrumbs.css

```diff
@@ -5,4 +5,13 @@
 
 .package-breadcrumbs .breadcrumbs-tag {
   @apply text-sm px-3 py-0.5 rounded font-bold text-white inline-block;
+}
+
+.learn-breadcrumbs li:not(:last-child)::after {
+  @apply text-lighter;
+  content: " >";
+}
```

Comment:
is that necessary? I thought the separator was an svg icon.

Reply:
My bad. Thats an error, forgot it in there while still using the `>` as seperator.